src/App.js

```diff
@@ -31,21 +42,132 @@ class App extends React.Component {
+    );
+  };
+
+  checkGuess = () => {
+    let temp = true;
+    for (let char of this.state.currWord) {
+      if (!this.state.guessedLetters.includes(char)) {
+        temp = false;
+      }
+    }
+
+    this.setState(
+      {
+        correctGuess: temp,
+      },
+      () => {
+        if (temp) {
+          this.countScore();
+        }
+      }
+    );
+  };
+
+  countScore = () => {
+    this.setState((prevState) => ({
+      scores: prevState.scores + 1,
+    }));
+  };
+
+  reset = () => {
+    this.setState({
+      currWord: getRandomWord(),
+      guessedLetters: [],
+      numGuesses: TOTALGUESS,
+      formInput: "",
+      correctGuess: false,
+    });
+  };
+
   render() {
+    const gameEnd = this.state.numGuesses === 0 || this.state.correctGuess;
+    const gameReset = (
+      <button className="btn btn-dark" onClick={this.reset}>
+        Reset
+      </button>
+    );
+    const imgSrc = require(`./PNG/${this.state.numGuesses}.png`);
+
     return (
-      <div className="App">
-        <header className="App-header">
-          <h1>Guess The Word 🚀</h1>
-          <h3>Word Display</h3>
-          {this.generateWordDisplay()}
-          <h3>Guessed Letters</h3>
-          {this.state.guessedLetters.length > 0
-            ? this.state.guessedLetters.toString()
-            : "-"}
-          <h3>Input</h3>
-          {/* Insert form element here */}
-          Todo: Insert form element here
-        </header>
+      <div className="container">
+        <div
+          className="row d-flex align-items-center"
+          style={{ minHeight: "100vh" }}
+        >
+          <div className="col-6">
+            <img
+              src={imgSrc}
+              alt={`Guesses left: ${this.state.numGuesses}`}
+              style={{ height: "400px" }}
+            />
+          </div>
+          <div className="col-6 ">
+            <h1>Guess The Word 🚀</h1>
+            <br />
+            <h1>{this.generateWordDisplay()}</h1>
+            <p>Number of guesses left: {this.state.numGuesses}</p>
+            <p>
+              Guessed Letters:{" "}
+              {this.state.guessedLetters.length > 0
+                ? this.state.guessedLetters.toString()
+                : "-"}
+            </p>
+            {/* Insert form element here */}
+            <form onSubmit={this.handleSubmit}>
+              <input
+                type="text"
+                value={this.state.formInput}
+                onChange={this.handleChange}
+                disabled={gameEnd}
+              />
+              <input type="submit" value="submit" disabled={gameEnd} />
+            </form>
+            <p>Overall score: {this.state.scores}</p>
+            {console.log(this.state.correctGuess)}
+            {this.state.correctGuess && (
+              <div>
+                <h1>Congrats!</h1>
+                {gameReset}
+              </div>
+            )}
+            {!this.state.correctGuess && this.state.numGuesses < 1 && (
```

Comment:
```suggestion
            {!this.state.correctGuess && !this.state.numGuesses && (
```

Alternatively we could try to store this combination in a variable and give it a meaningful meaning to make it easily understood.
e.g. `const gameOver = !this.state.correctGuess && !this.state.numGuesses` - like that anyone would know what this combination symbolizes